tests/Dockerfile.debian-bullseye

```diff
@@ -27,6 +27,15 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y \
     acl \
     build-essential \
+    gcc-aarch64-linux-gnu \
```

Comment:
To avoid slowing down the docker build process of these already large images, we may need to move these tool-chain installations to a separate "RUN" instruction and apply some conditional-fetching logic to avoid going through these steps in the regular/common case (non-cross-compilation scenarios).

```
RUN if [ "$ARCH" = "HOST_ARCH" ]; then \
     apt-get install libc6-dev-arm64-cross ...
```

That was just fyi, we can do this ourselves later on.

Reply:
You'll need to make ARCH an ARG to ensure it rebuilds. I didn't do it that way because the cross compile tool chain install was quick compared to the overall build times of the image. Making it $ARCH based would extend the total build times by alot, but I never checked the image size.

If you use the ARG approach and move it toward the bottom then the build time should stay low and reduce the non-cross image size. Not sure why I didn't think of that 😆